projects/sandbox/infer/infer/callback.py

```diff
@@ -116,8 +119,9 @@ def cluster(self, y) -> TimeSlideEventSet:
         # subtract off the time required for
         # the coalescence to exit the filter
         # padding and enter the input kernel
-        # to the neural network
-        t0 = self.start - self.fduration / 2
+        # to the neural network, and account
+        # for the PSD burn-in that has been removed
+        t0 = self.start - self.fduration / 2 + self.psd_length
```

Comment:
Should `psd_length` be subtracted here? Seems like it should shift things in the same direction as `fduration`, though I'm having a hard time thinking this through.